lib/run-script.js

```diff
@@ -84,31 +84,52 @@ function runScript (args, cb) {
 
 function list(cb) {
   var json = path.join(npm.localPrefix, "package.json")
+  var cmdList = [ "publish", "install", "uninstall"
```

Comment:
For bonus points, something that abstracted this list of lifecycle scripts out into a separate function would be useful, instead of having this scattered around in a few different places.